packages/block-library/src/heading/block.json

```diff
@@ -14,7 +14,7 @@
 		"content": {
 			"type": "string",
 			"source": "html",
-			"selector": "h1,h2,h3,h4,h5,h6",
+			"selector": "h2",
```

Comment:
I assume it was added for testing purposes and it will get reverted.

Reply:
Yep! The processor API doesn't yet support arbitrary CSS selector.